src/main/java/appeng/tile/networking/CableBusTESR.java

```diff
@@ -21,14 +21,16 @@
 
 import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;
 import net.minecraft.util.EnumFacing;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
 
 import appeng.api.parts.IPart;
 import appeng.tile.AEBaseTile;
 
 
+@SideOnly( Side.CLIENT )
 public class CableBusTESR extends TileEntitySpecialRenderer<AEBaseTile>
 {
-
 	@Override
 	public void render( AEBaseTile te, double x, double y, double z, float partialTicks, int destroyStage, float p_render_10_ )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)